ONSdigital/response-operations-social-ui
# Redirect and flash message on generate IAC

### Context
The generate IAC button was rendering the next page on the POST method, this meant refreshing the next page could repeatedly generate more IAC's unintentionally. This PR changes the generate IAC endpoint to redirect to back to the same case details route and store the IAC in a flashed message, so that it is only displayed back once.

### What Has Changed
* Changed generate IAC endpoint to flash message the IAC and redirect
* Added code to pop and display the IAC from the flashed messages if present on the case details page

### How to Test
Go to a case details page and generate a new IAC. You should be directed back to the same url but with the new iac displaying, and it should disappear on a page refresh without generating more codes.

### Links
https://trello.com/c/2r4ls2vs/293-if-generate-new-hac-then-refresh-the-page-a-new-hac-is-generated
